assets/ui-icons/index.ts

```diff
@@ -21,6 +21,7 @@ export { default as BinIcon } from './kos.svg'
 export { default as CrossIcon } from './krizik.svg'
 export { default as SearchIcon } from './lupa.svg'
 export { default as MailIcon } from './mail.svg'
+export { default as MapPin } from './map-pin.svg'
```

Comment:
`ui-icons` folder is meant just for UI icons (standard 24x24 icons, something like our own version of material icons).
These icons are sed in bratislava design system (bratislava.sk, konto.bratislava.sk...), but finally, we decided to use material icon in Paas MPA, so this folder will probably be removed from repo.

Please create new folder (`map`?) in assets and add pins there (alog with barrel file (index.ts))